src/test/java/testCases/Validate_ResourcesPage.java

```diff
@@ -0,0 +1,77 @@
+//Test case to check error validation of each field on the Financial Assessment page
+package testCases;
+
+import org.testng.annotations.Test;
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import utility.Constant;
+import appModules.Navigation_Action;
+import appModules.Resources_Action;
+import appModules.Trading_Action;
+
+public class Validate_ResourcesPage {
+    public WebDriver driver;
+    //Test Method to login into the site
```

Comment:
Use description in the annotation rather than comments. Please have a look at other tests.